package.json

```
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "ts-node index.ts",
+    "start": "npm run testconfig && ts-node index.ts",
```

Pour faire genre tu maîtrise NPM
```suggestion
    "prestart": "npm run testconfig",
    "start": "ts-node index.ts",
```